apps/antalmanac/src/components/AppBar/Export.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Paper, Popover, Theme } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { ClassNameMap, Styles } from '@material-ui/core/styles/withStyles';
+import { IosShare } from '@mui/icons-material';
+import { PureComponent } from 'react';
+
+import AppStore from '$stores/AppStore';
+import ExportCalendar from '$components/Calendar/Toolbar/ExportCalendar';
+import ScreenshotButton from '$components/Calendar/Toolbar/ScreenshotButton';
+
+const styles: Styles<Theme, object> = {
+    container: {
+        padding: '0.75rem',
+        minWidth: '12.25rem',
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.5rem',
+    },
+    betaBadge: { transform: 'scale(1) translate(95%, -50%)' },
+};
+
+interface SettingsState {
+    anchorEl?: HTMLElement;
+    theme: string;
+    screenshotting: boolean;
```

Comment:
Why do we need to store this `screenshotting` state? Where are we using it?

Reply:
Yeah, I genuinely have no recollection of writing this. It is very very strange code. I'm obliterating it soon.